src/pages/DeleteCar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Cars from '../cars.json';
+import '../assets/App.css';
+
+const DeleteCar = () => (
+  <div className="content">
+    <h2>Reservations page</h2>
+    <div className="reserved-cars">
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[0].image} alt={Cars[0].id} />
+        </div>
+        <p>{Cars[0].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[1].image} alt={Cars[1].id} />
+        </div>
+        <p>{Cars[1].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[2].image} alt={Cars[2].id} />
+        </div>
+        <p>{Cars[2].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[0].image} alt={Cars[0].id} />
+        </div>
+        <p>{Cars[0].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[1].image} alt={Cars[1].id} />
+        </div>
+        <p>{Cars[1].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+      <div className="r-car">
+        <div className="img-container">
+          <span className="dot" />
+          <img src={Cars[2].image} alt={Cars[2].id} />
+        </div>
+        <p>{Cars[2].model}</p>
+        <button type="button" className="button-delete">Delete</button>
+      </div>
+    </div>
+  </div>
+);
+
+export default DeleteCar;
```

Comment:
[Optional] - I suggest not focusing on deleting the car for now. Because in reality, deleting will be just a simple function in the redux reducer where we don't need to display any data for the UI. Just a click is enough to delete the car and navigate or reload the page.

Reply:
Hey @rex-9 , I saw a that on our main page there is a link to delete items, which is specified on the screenshot below, so I thought it needs its own page implemented, what do you think about it?

![image](https://user-images.githubusercontent.com/71190715/204108126-baa3525a-e523-413b-ad5c-539447cf248c.png)